packages/core/src/application.ts

```diff
@@ -59,6 +59,8 @@ export class Application extends Context {
 
     this.handleHttp = (req: ServerRequest, res: ServerResponse) =>
       this._handleHttpRequest(req, res);
+
+    this.bind('logError').to(this._logError.bind(this));
```

Comment:
>Should we name it as something like loggers:error-logger?

I think pretty soon, we should figure something out for `find-route` and `invoke-method` too. Not sure what namespace though (`routing:find-route`, `sequence:invoke-method`?). Maybe discuss during next sync up.